docs/_docs/reference/other-new-features/opaques-details.md

```diff
@@ -51,6 +51,8 @@ object o:
 def id(x: o.T): o.T = x
 ```
 
+Opaque type aliases cannot be `private` and cannot be overridden in subclasses.
```

Comment:
I think we should point out that `private[OuterScope]` is allowed however.
Another thing that might not be very clear here is that one can define an abstract type and then override (or implement) it with an opaque type

Reply:
I think that could be elsewhere, maybe in a tutorial. The doc here is a spec and conciseness is good. If we say however it can be private[X] the we'd have to go to all _other_ places where we say `private` and include or exclude private `X`.

Reply:
If it's consistent across the docs that `private[Scope]` is not `private` then it's OK